packages/dbml-core/package.json

```diff
@@ -34,9 +34,9 @@
   "dependencies": {
```

Comment:
Should add build script for your `dbmlparserv2` here so that it get build automatically before dbml/core